README.md

```diff
@@ -82,65 +51,47 @@ allows to encode more data in the markup like e.g. the post's id:
 </article>
 ```
```

Comment:
Would be cool to add an example for a component with some `data-test-*` attributes, e.g.:

```hbs
{{comments-list data-test-comments-for=post.id}}
```

or so